question.html

```diff
@@ -98,63 +98,75 @@ <h1 style="font-family: 'Bubblegum Sans', cursive;"><b>Let's ASK!</b></h1>
 	</span>
 	</form>
 </div>
-
-<div class="container">
```

Comment:
When you search for a question and there are no hits, the container should fully disappear.
Somehow, the box remains:

![bildschirmfoto 2015-11-02 um 22 54 57](https://cloud.githubusercontent.com/assets/1174752/10884786/ec73fe7e-81b4-11e5-821c-9e74f3de080f.png)

Please create an issue for this.